gem/optimise.py

```diff
@@ -828,6 +826,14 @@ def arg_ind_flat(self):
     def arg_ind_set(self):
         return set(self.arg_ind_flat)
 
+    def copy(self):
+        new_rep = list(self.rep)
+        for new_summand, summand in zip(new_rep, self.rep):
+            new_summand = Summand(summand)
+            for k, v in iteritems(summand):
+                new_summand[k] = list(v)
+        return LoopOptimiser(rep=new_rep, multiindex=self.multiindex, arg_ind=self.arg_ind)
```

Comment:
This method is not used anywhere, it's only for testing earlier, will delete soon